src/app/core/admin/feedback/admin-list-feedback.component.ts

```diff
@@ -69,11 +153,48 @@ export class AdminListFeedbackComponent extends AbstractPageableDataComponent<an
 			});
 	}
 
+	getDefaultQuickFilters() {
+		return { open: { show: false, display: 'New/Open Issues' } };
+	}
+
+	getQuery(): any {
+		let query: any;
+
+		if (this.filters.open.show) {
+			query = {
+				$or: [{ status: FeedbackStatusOption.NEW }, { status: FeedbackStatusOption.OPEN }]
```

Comment:
@sshah-asymmetrik can this be `status: { $not: FeedbackStatusOption.CLOSED }` in case we want to add other states (like "Pending Customer" or otherwise)? Any issues there?